CONTRIBUTING.md

```diff
@@ -51,14 +51,20 @@ We use and recommend the following workflow:
 3. Create a branch off of master (`git checkout -b mybranch`).
     - Name the branch so that it clearly communicates your intentions, such as issue-123 or githubhandle-issue.
     - Branches are useful since they isolate your changes from incoming changes from upstream. They also enable you to create multiple PRs from the same fork.
-4. Make and commit your changes.
+4. Make and commit your changes to your branch
     - Please follow our [Commit Messages](#commit-messages) guidance.
 5. Add new tests corresponding to your change, if applicable.
 6. Build the repository with your changes.
     - Make sure that the builds are clean.
     - Make sure that the tests are all passing, including your new tests.
-7. Create a pull request (PR) against the upstream repository's **master** branch.
-    - Push your changes to your fork on GitHub (if you haven't already).
+7. Create a pull request (PR) against the dotnet/runtime repository's **master** branch.
+    - State in the description what issue or improvement your change is addressing.
+    - Check if all the Continuous Integration checks are passing.
+8. Wait for area owners feedback or approval of your changes.
+    - Details about the pull request [review procedure](doc/pr-guide.md)
+9. You change is in!
```

Comment:
```suggestion
9. When Microsoft employee(s) have signed off, and all checks are green, your change will be merged.
```

Reply:
Did you mean committer(s) or reviewer(s)?

Reply:
Community reviews are welcome but I do not think we will merge something unless someone with commit rights has reviewed it.

Almost all area owners have commit rights. But we do not require an area owner to sign off to the change -- it is based on best judgement about the change.

I"ll let you pick the words though 👍